specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-06-01-preview/cluster.json

```diff
@@ -0,0 +1,1538 @@
+{
```

Comment:
For new API versions, we much prefer that the first commit be an unchanged copy of the last reviewed API version. Then subsequent commits can reflect the necessary changes. This facilitates simpler review of the changes in GH.

Reply:
I see - thanks, that makes a lot of senses. We will update the doc on our end to add this step for any new API changes going forward.